recipes/libdc1394/all/conanfile.py

```diff
@@ -13,42 +16,61 @@ class Libdc1394Conan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
     options = {"shared": [True, False], "fPIC": [True, False]}
     default_options = {"shared": False, "fPIC": True}
+
+    generators = "pkg_config"
     _env_build = None
 
     @property
     def _source_subfolder(self):
         return "source_subfolder"
 
-    def _configure_autotools(self):
-        if not self._env_build:
-            self._env_build = AutoToolsBuildEnvironment(self)
-            if self.options.shared:
-                args = ["--disable-static", "--enable-shared"]
-            else:
-                args = ["--disable-shared", "--enable-static"]
-            args.extend(["--disable-examples"])
-            self._env_build.configure(args=args)
-        return self._env_build
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
 
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
-        if self.settings.os == "Windows":
-            raise ConanInvalidConfiguration("Windows is not supported")
-        if self.settings.compiler == "clang":
-            raise ConanInvalidConfiguration("Clang doesn't support VLA")
         del self.settings.compiler.libcxx
         del self.settings.compiler.cppstd
 
-    def config_options(self):
+    def requirements(self):
+        self.requires("libusb/1.0.24")
```

Comment:
libusb is an optional dependency, but there is no easy way to guarantee that a system libusb won't be found (in case of cross-build to iOS, test package of libdc1394 failed with undefined references to libusb).

Reply:
The easiest way with autoconf is to use [`AC_ARG_WITH`](https://www.gnu.org/software/autoconf/manual/autoconf-2.69/html_node/External-Software.html#External-Software) + some bash checks.

The easy was ironic :smile: